packages/features/bookings/Booker/utils/event.ts

```diff
@@ -110,8 +110,32 @@ export const useScheduleForEvent = ({
     isTeamEvent,
     orgSlug,
     teamMemberEmail,
+    useApiV2: true,
```

Comment:
@ThyMinimalDev IMO this should be behind a feature flag so that we can turn it off easily in case of failure reports